scene/gui/texture_button.cpp

```diff
@@ -124,6 +124,10 @@ void TextureButton::_notification(int p_what) {
 			DrawMode draw_mode = get_draw_mode();
 
 			Ref<Texture2D> texdraw;
+			Ref<BitMap> texdraw_bitmap;
```

Comment:
What's the purpose of this variable if it's never re-assigned? Why not just use `click_mask`?

Reply:
Because its used to get size, its meaning as "click_mask" is not relevant in this context. Its only used as `BitMap` to fallback to, to get button region size. I've created it because I couldnt cast Bitmap to Texture2D, nor find common general type. So I've decided to logically link it to `Ref<Texture2D> texdraw;`. Mostly for clarity's sake, not necessary.

Reply:
If it's not necessary then remove it. It doesn't help clarity that much.

Reply:
Also thought that it might be assigned some other bitmap, not only click_mask, to fallback to some other object

Reply:
When that happens the new variable can be added. Right now it's redundant.

Reply:
ok, removed